Sources/Models/GitHub.swift

```diff
@@ -39,18 +40,21 @@ func githubRepository(from url: URL) -> Result<GitHubRepository, GeneratePlistEr
 
     guard let host = url.host,
         host.contains(gitDomain),
-        url.pathComponents.count >= 2,
-        url.pathComponents[url.pathComponents.count - 1].hasSuffix(gitSuffix) else { return .failure(.unknownRepository) }
+        url.pathComponents.count >= 2
+    else { return .failure(.unknownRepository) }
 
+    let name = url.pathComponents.last!.contains(gitSuffix)
```

Comment:
Even if the .count >= 2, please don't force unwrap, instead make `let lastPathComponent = url.pathComponents.last` in the `guard` condition.